lib/bios/menus.robot

```diff
@@ -410,11 +410,23 @@ Reset To Defaults Tianocore
     [Documentation]    Resets all Tianocore options to defaults. It is invoked
     ...    by pressing F9 and confirming with 'y' when in option
     ...    setting menu.
+    Power On
```

Comment:
This keyword is always used when we are already in the UEFI setup, right?

Do we have to use hard power cycle, can we instead issue reset?

Reply:
I suppose yes. Changed here: https://github.com/Dasharo/open-source-firmware-validation/compare/4364e09f38fab8ea9bca2f303d4b42a33af0280b..a0485cccb7f9f1c6c2c70ff4763da0b2ad0ae52e
Will update with results of a quick test if it does indeed work as well.